ur_description/CMakeLists.txt

```
@@ -1,153 +1,27 @@
 cmake_minimum_required(VERSION 2.8.3)
```

```suggestion
cmake_minimum_required(VERSION 3.5.2)
```

Since you're here...